docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,822 @@
       },
     ]}
   />,
 );
 
 card(
-  <Example
-    name="Tooltips"
-    description="General Tooltip usage involves wrapping your target in a Tooltip. Notice how long
-    text is wrapped around underneath."
-    defaultCode={`
-<Tooltip inline text="Logout">
-  <Icon
-    accessibilityLabel="Logout of your profile"
-    color="pine"
-    icon="logout"
+  <MainSection name="Best practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use a Tooltip to describe the function of an interactive element, typically an [Icon Button](/IconButton), in as few words as possible."
+        defaultCode={`
+<Tooltip text="Send Pin">
+  <IconButton
+    accessibilityLabel=""
+    bgColor="white"
+    icon="share"
+    iconColor="darkGray"
+    size="lg"
   />
+</Tooltip>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Use a Tooltip to restate text already visible on screen."
+        defaultCode={`
+<Tooltip text="Save">
+  <Button color="red" text="Save" size="lg" inline />
 </Tooltip>
 `}
-  />,
-);
-
-card(
-  <Example
-    name="Tooltips with link"
-    description="Passing in a link will render the link in the tooltip"
-    defaultCode={`
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use Tooltip to distinguish related actions with visually similar iconography."
+        defaultCode={`
+<Flex>
+  <Tooltip text="Align left">
+    <IconButton
+      accessibilityLabel=""
+      bgColor="white"
+      icon="text-align-left"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+  <Tooltip text="Align center">
+    <IconButton
+      accessibilityLabel=""
+      bgColor="white"
+      icon="text-align-center"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+  <Tooltip text="Align right">
+    <IconButton
+      accessibilityLabel="Align right"
+      bgColor="white"
+      icon="text-align-right"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+</Flex>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Use Tooltip to communicate critical information, such as an error, instructions for performing a task, or interaction feedback."
+        defaultCode={`
+<Tooltip text="Pssst! Looks like you've already saved this Pin.">
+  <Button color="red" text="Save" inline size="lg" />
+</Tooltip>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use Tooltip to add supplementary information about a feature, typically paired with an `info-circle` [IconButton](/IconButton)."
+        defaultCode={`
+<Tooltip inline text="Total ad spend in the select time period">
+  <Icon
```

Comment:
This should be an IconButton, needs to tab-able

Reply:
DONE